src/Microsoft.Framework.ApplicationHost2/project.json

```diff
@@ -50,6 +47,7 @@
                 "System.Reflection.Extensions": "4.0.0-beta-*",
                 "System.Runtime": "4.0.20-beta-*",
                 "System.Runtime.Extensions": "4.0.10-beta-*",
+                "System.Runtime.Loader": "4.0.0-beta-*",
```

Comment:
What should it be? It's needed to get the Assembly Version in ReferenceAssemblyDependencyResolver:

```
internal static NuGetVersion GetAssemblyVersion(string path)
{
#if ASPNET50
    return new NuGetVersion(AssemblyName.GetAssemblyName(path).Version);
#else
    return new NuGetVersion(System.Runtime.Loader.AssemblyLoadContext.GetAssemblyName(path).Version);
#endif
}
```